### ISSUE TITLE
Image mode in slideshow

### ISSUE THREAD

**Issue by shifuren** [lumien231/Custom-Main-Menu#76]
So, is there any reason "mode" : "stretch" or "mode" : "fill" doesn't work in slideshow mode? Images are basically stuck to one by default :(

**Comment by lumien231** [comment]
So would the stretch / fill to the defined width & height or what?

**Comment by shifuren** [comment]
Just want cycling in slideshow mode through images created by stretch and fill mode, right now slideshow mode does some resizing by default, not letting to specify it

**Comment by lumien231** [comment]
Ok, will look into it. But what do you mean with blur? Slideshows don't have blur to begin with.

**Comment by shifuren** [comment]
Blur is just a feature request, if it is doable, maybe will be useful to some of us

**Comment by lumien231** [comment]
Yeah but you could also just add blur to the images youself (Really you could also change the size of the images yourself making the mode obsolete)
